logs-shipper/src/main/java/jersey/rest/SearchResource.java

```diff
@@ -10,45 +13,55 @@
 import org.elasticsearch.search.SearchHit;
 import org.elasticsearch.search.SearchHits;
 import org.elasticsearch.search.builder.SearchSourceBuilder;
+import pojo.Account;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
+import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.io.IOException;
 import java.net.HttpURLConnection;
+import java.util.Optional;
 
 import static java.util.Objects.requireNonNull;
 
 @Singleton
 @Path("/")
 public class SearchResource {
 
-    private static final String INDEX = "posts";
-
     private final RestHighLevelClient elasticSearchClient;
+    private final AccountsServiceApi accountsServiceApi;
 
     @Inject
     public SearchResource(RestHighLevelClient elasticSearchClient) {
         this.elasticSearchClient = requireNonNull(elasticSearchClient);
+        this.accountsServiceApi = new AccountsServiceApi();
     }
 
     @GET
     @Path("/api/search")
     @Produces(MediaType.APPLICATION_JSON)
-    public Response search(@QueryParam("message") String message, @QueryParam("header") String header) {
+    public Response search(@HeaderParam ("X-ACCOUNT-TOKEN") String tokenHeader,
```

Comment:
Let's have `X-ACCOUNT-TOKEN` somewhere as a constant (since the code using it in multiple places)